packages/components/src/components/autocomplete/autocomplete.tsx

```diff
@@ -1,6 +1,7 @@
-import classNames from 'classnames';
 import React from 'react';
+import classNames from 'classnames';
 import { matchStringByChar, getPosition } from '@deriv/shared';
+import { localize } from '@deriv/translations';
```

Comment:
We should found another way for this change, there should not be a single connection between Components and Translation packages.